test/test_for_each.cpp

```diff
@@ -96,7 +96,7 @@ THRUST_HIP_LAUNCH_BOUNDS_DEFAULT
 void simple_test_kernel(F func, int size)
 {
     // (void) func; (void) size;
-    thrust::for_each(thrust::seq,
+    thrust::for_each(thrust::hip::par,
```

Comment:
What is the purpose of this change?

Reply:
When we call ```thrust::hip::par``` on device side it call the thrust::seq algorithm. So with this we increased the test coverage, because we check the device side handling of the ```thrust::hip::par``` case too.

Reply:
But we removed thrust:seq as a test case. Are we ok to drop that test?

Reply:
Wrote out the issue.